lib/config.js

```diff
@@ -85,55 +85,27 @@ module.exports = {
       'discord.gg',
       'twitch.tv'
     ],
-    // List of filtered swear words to auto-delete (from http://www.bannedwordlist.com/), these words are checked against word boundaries
+    // List of filtered swear words to auto-delete (selected from https://en.wikipedia.org/wiki/List_of_ethnic_slurs), these words are checked against word boundaries
     filteredSwearWords: [
-      'anal',
-      'anus',
-      'bitch',
-      'blowjob',
-      'blow job',
-      'bollock',
-      'bollok',
-      'boner',
-      'boob',
-      'butt',
-      'buttplug',
-      'clitoris',
-      'cock',
+      'chink',
       'coon',
-      'cunt',
-      'dick',
-      'dildo',
       'dyke',
       'fag',
       'faggot',
-      'fellate',
-      'fellatio',
-      'felching',
-      'fudgepacker',
-      'fudge packer',
-      'flange',
       'homo',
       'jizz',
-      'knobend',
-      'knob end',
-      'labia',
-      'muff',
+      'nazi',
```

Comment:
Is this usually that offensive?

Reply:
A bit of a mixed bag, I guess. In context of RL Discord, I think limiting it would be used more as an insult than anything. Plus I'd steer clear of any political discussion anyway.